### ISSUE TITLE
Identifier les facteurs de qualité et d'exemplarité des projets

### ISSUE THREAD

**Issue by iolyd** [CUPUM/nplex-frontend#27]
L'évaluation de la qualité et de l'exemplarité des projets nécessite d'abord d'établir une définition de l'exemplarité et d'en construire une charte ou  une grille d'évaluation.

### Les retours du comité de conseil

Dans l'atelier 1, les membres du CC seront invités à élaborer leur interprétation de l'exemplarité.

#### L'agenda de la Ville

Ces textes peuvent donner une idée de certains critères d'évaluation utilisés par le milieu preneur et donc à prendre en considération:

- https://www.realisonsmtl.ca/qualite
- https://designmontreal.com/sites/designmontreal.com/files/publications/agenda_mtl_2030_v1.12-19_fr_lr.pdf

#### Le système de Davos pour l'évaluation de la qualité

Ce système est entre autres utilisé par diverses institutions au Québec.

- https://www.bak.admin.ch/bak/fr/home/baukultur/qualitaet/davos-qualitaetssystem-baukultur.html#:~:text=Il%20s'agit%20d'un,Esprit%20du%20lieu%20et%20Beaut%C3%A9.

## Modèles / synthèse

| Critère | Description | Source(s) | Proposition d'intégration |
| --- | --- | --- | --- |

**Comment by iolyd** [comment]
- [ ] Simplifier et réorganiser
- [ ] Inclure les constats du rapport de Davos

**Comment by AgathePC** [comment]
@iolyd J'ai fait une première version. Nous pourrons en discuter afin d'ajouter des corrections ou de faire une nouvelles version. Le document de travail est lié dans le Issue.

**Comment by iolyd** [comment]
@AgathePC Super. Je ne serai probablement pas disponible ce vendredi, je propose donc lundi prochain pm en personne.

**Comment by AgathePC** [comment]
@iolyd Parfait c'est noté

**Comment by iolyd** [comment]
@AgathePC Oui, voici un lien OneDrive: https://udemontreal-my.sharepoint.com/:b:/g/personal/emmanuel_beaudry_marchand_umontreal_ca/EfxGtDdT7wdDi8ONDjG_tMQB7qbCL4AmIuhLCd1lnhMFaA?e=KaSswG

**Comment by iolyd** [comment]
Related to some points discussed in #169
